pyspectral/radiance_tb_conversion.py

```diff
@@ -308,7 +309,7 @@ def radiance2tb(self, rad):
         beta = SEVIRI[self.bandname][self.platform_name][2]
 
         tb_ = c_2 * vc_ / \
-            (alpha * np.log(c_1 * vc_ ** 3 / rad + 1)) - beta / alpha
+            (alpha * da.log(c_1 * vc_ ** 3 / rad + 1)) - beta / alpha
```

Comment:
Maybe we should just check how the cached-LUT files look like now, when we do not use masked arrays?
` np.savez(filepath, tb=tb_, radiance=rad)`